lib/matplotlib/backend_bases.py

```diff
@@ -2763,6 +2763,10 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
         """Draw a rectangle rubberband to indicate zoom limits"""
         pass
 
+    def remove_rubberband(self):
```

Comment:
Not at this time. I found a class named `RubberbandBase` in backend_tools.py that looks like a template for rubberband functionality, including a `remove_rubberband` function. Gtk3 and TK seem to implement this class. However, this code path is never used. Seems to me that this solution has been abandoned at some time and now only `draw_rubberband` implemented by the canvas is used.